HTML/syntax_test_html.html

```diff
@@ -409,7 +409,7 @@
         <style
         type
         =
-        text/csss>
+        text/css_>
```

Comment:
I tend to suggest using a `_` suffix to indicate "like this word, but intentionally not this word" as opposed to misspelling a word, it's too hard to otherwise distinguish intentional from accidental typos. (The vast majority of the items in this PR are accidental, although there are a couple that appear to be intentional.)

Reply:
Yes. This change was surprising until I saw the assertion below. Your convention seems logical.